### ISSUE TITLE
When going to the alarms setting of the app, get the alarms already set in the device first

### ISSUE THREAD

**Issue by mousse04** [piggz/harbour-amazfish#19]
When you set up an alarm, the best is to get the alarms already set on the device, display them in the app and then let user add, modify, remove an existing alarm or create a new one.

**Comment by piggz** [comment]
I think this is now fixed.  Can you confirm?

**Comment by piggz** [comment]
I dont think it updates any modified alarm times, but it should update the enabled/disabled status.

**Comment by mousse04** [comment]
Hi Adam,
I've tried recently and this doesn't seem to be fixed. 
Maybe I was not clear enough. 
The scenario is : 
Imagine you have 2 devices, one Android and one with SFOS installed. If you use your Android device and set up some internal alarm within the band. When you switch to the SFOS device , you may expect that the alarm set within the band are read by your app, and displayed so that you know what is defined already. 
Does it make sense ? 
Thanks

**Comment by piggz** [comment]
Can you check this on current builds?
